src/schema/v2/me/myCollection.ts

```diff
@@ -67,13 +67,23 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
         },
       }),
     },
+    excludePurchasedArtworks: {
+      type: GraphQLBoolean,
```

Comment:
@MounirDhahri feel free to commit and add a default value here:

```suggestion
      type: GraphQLBoolean,
      defaultValue: false,
```

Reply:
🙏